microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2136,7 +2134,7 @@ Receiver:
 <!-- ====================================================================== -->
 ## Managing SmartScreen
 
-<!-- todo: move this section into the above outline structure.  if this is h2, add a "Managing SmartScreen" row to the nav table at top. -->
+<!-- 112 todo: move this section into the above outline structure.  if this is h2, add a "Managing SmartScreen" row to the nav table at top. -->
```

Comment:
in this 112 PR, move this "Managing SmartScreen" up into the above outline structure, maybe add table row.

Reply:
AFAIK [SmartScreen](https://learn.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-overview) is a Windows reputation check product name. ReputationCheck is the API name we decided on during API review. I believe this IsReputationCheckingRequired controls whether SmartScreen is turned on. @nishitha-burman @novac42 please help advise.

Reply:
Renamed "Tracking Prevention" section to "Security and Privacy", added the incoming link there as its own tab-set, deleted "Managing SmartScreen" section.